dials_data/definitions/l_cysteine_dials_output.yml

```diff
@@ -1,22 +1,68 @@
 name: DIALS output of processing an L-cysteine dataset
-author: Ben Williams (2018)
+author: Ben Williams (2019)
 license: CC-BY 4.0
 description: >
   Typical DIALS output for a small-molecule dataset.
 
-  These were derived from the DIALS small molecule tutorial L-cysteine dataset
-  (https://zenodo.org/record/51405 files l-cyst_0[1-4].tar.gz).
+  Includes ten image files for each of four sweeps (a subset of https://doi.org/10.5281/zenodo.51405,
+  l-cyst_0[1-4].tar.gz, uncompressed) and associated processed files.
 
   On a linux terminal, the relevant CBF-format image files can be obtained with
```

Comment:
would probably remove those three lines when images are included in the dataset.